agent/acs/handler/acs_handler_test.go

```diff
@@ -518,11 +518,14 @@ func TestHandlerReconnectDelayForInactiveInstanceError(t *testing.T) {
 		}).Return(fmt.Errorf("InactiveInstanceException:")),
 		mockWsClient.EXPECT().Connect().Do(func() {
 			reconnectDelay := time.Now().Sub(firstConnectionAttemptTime)
+			reconnectDelayTime := time.Now()
 			t.Logf("Delay between successive connections: %v", reconnectDelay)
-			// On window platform, we found issue with time.Now().Sub(...) reporting 199.9989 even after the code has already waited for time.NewTimer(200)ms.
 			timeSubFuncSlopAllowed := 1 * time.Millisecond
-			if reconnectDelay < inactiveInstanceReconnectDelay-timeSubFuncSlopAllowed {
-				t.Errorf("Reconnect delay %v is 1 ms less than inactive instance reconnect delay %v", reconnectDelay, inactiveInstanceReconnectDelay)
+			// On window platform, we found issue with time.Now().Sub(...) reporting 199.9989 even after the code has already waited for time.NewTimer(200)ms.
+			if reconnectDelay < inactiveInstanceReconnectDelay {
```

Comment:
There are 2 conditions here
1) to make sure reconnectDelay is "greater" than inactiveInstanceReconnectDelay
2) or to make sure it is within duration of the 1 milisecond, if reconnectDelay is "less" than inactiveinstanceReconnectDelay
The code here is intended for "1"